test/JDBC/upgrade/latest/schedule

```diff
@@ -314,6 +314,7 @@ babel_sqlvariant_cast_compare
 BABEL-3144
 sys-all_parameters-dep
 BABEL-3556
+BABEL-3588
```

Comment:
To verify the upgrade script change, please add this to other schedule files such as 13_4, 14_5, etc.. Note that `latest` schedule file does not execute any upgrade script.